syncstorage/views/decorators.py

```diff
@@ -28,6 +28,10 @@
 # How long the client should wait before retrying a conflicting write.
 RETRY_AFTER = 10
 
+# How long a client should wait to retry. It won't succeed, but should
+# return a 4xx error punting them to the new server.
```

Comment:
As I Understand, when a client gets a 4xx error, it needs to get a new token because it's been "reassigned". The migration script reaches into the tokenserver db as well to "assign" the user to spanner. I was originally thinking that I'd just wait until the client's token expires and the client naturally goes and checks for the new assignment, since that would potentially give the longest migration period.  That said, going to add a column to `migration` that is the `state`.

Reply:
Gotcha. I think it would be OK to do as you suggest and just wait for the token to naturally expire, if you don't want the additional complexity (but in that case we should change the comment above to avoid confusion).

Reply:
Although do be aware that we can under some circumstances accept expired tokens for up to 2 hours after they expire, for reasons:

https://github.com/mozilla-services/server-syncstorage/blob/a7649edc1f02e529beb85a24c329f1c5ce108f4d/syncstorage/views/authentication.py#L18

I'm not sure we ever actually measured whether that was worth it in practice, we could consider removing it if we want users to hurry up and discover their migrated status.

Reply:
Yeah, totally fine with changing the comment. The code is already weirdly complex because of the fact that it's loosely bound between two packages. QA might be a bit upset that they have to wait.